bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/client/LivisiClient.java

```diff
@@ -32,10 +32,7 @@
 import org.eclipse.jetty.http.HttpStatus;
 import org.openhab.binding.livisismarthome.internal.LivisiBindingConstants;
 import org.openhab.binding.livisismarthome.internal.client.api.entity.StatusResponseDTO;
-import org.openhab.binding.livisismarthome.internal.client.api.entity.action.ActionDTO;
-import org.openhab.binding.livisismarthome.internal.client.api.entity.action.ShutterActionDTO;
-import org.openhab.binding.livisismarthome.internal.client.api.entity.action.ShutterActionType;
-import org.openhab.binding.livisismarthome.internal.client.api.entity.action.StateActionSetterDTO;
+import org.openhab.binding.livisismarthome.internal.client.api.entity.action.*;
```

Comment:
Star imports should only be used for static imports. This also resulted in a checkstyle warning: `AvoidStarImportCheck`. Please have a look at target/code-analysis/report.html, since there are a few others as well.

Reply:
done